app/scripts/tmx/tmx_mozillaorg

```diff
@@ -44,6 +44,13 @@ foreach (Files::getFoldersInPath(GIT . 'mozilla_org/') as $locale) {
 
         foreach ($strings as $english => $translation) {
             $output = function ($str1, $str2) use ($file, $english, $translation) {
+                if (Strings::endsWith(strtolower($str2), '{ok}')) {
```

Comment:
Since trailing and leading whitespaces are non significant in LANG files, can we just strip it and trim?

```PHP
if (Strings::endsWith(strtolower($str2), '{ok}')) {
    $str2 = trim(str_ireplace('{ok}', '', $str2));
}
```

Reply:
Or even simpler

```php
if (Strings::endsWith(strtolower($str2), '{ok}')) {
    $str2 = trim(rtrim($str2, '{ok}'));
}
```

Reply:
Hm, without stripping the trailing space, this query doesn’t return the same result (no perfect match for the locale) Do we care?
http://localhost:8082/?recherche=%3Ca+href%3D%22%25(mailto_inclusion)s%22%3Einclusion%40mozilla.com%3C%2Fa%3E&repo=mozilla_org&sourcelocale=en-US&locale=fr&search_type=strings&entire_string=entire_string

Reply:
oh, just noticed the trim, nevermind